ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -84,18 +84,30 @@ function BlockaidBannerAlert({ txData, ...props }) {
     reason,
     result_type: resultType,
   } = securityAlertResponse;
+
+  let title, description;
   if (resultType === BlockaidResultType.Benign) {
     return null;
+  } else if (resultType === BlockaidResultType.Warning) {
+    // When `result_type` is warning, the reason should no longer be relevant for
+    // determining the copy and we should always use the new copy. The reason for
+    // that is because Blockaid has lower certainty when they flag something as
+    // warning so that requires a softer and broader message than the ones we use
+    // when `result_type` is `Malicious` or `Error`.
+
+    title = t(REASON_TO_TITLE_TKEY[BlockaidReason.errored]);
+    description = t(REASON_TO_DESCRIPTION_TKEY[BlockaidReason.errored]);
```

Comment:
To clarify, is this the `Because of an error, we couldn't check for security alerts...` ?

How then do we differentiate between an actual error preventing validation, versus them "successfully" returning a warning response?

Reply:
Adjusted the copy, thanks both